Source/Fuse.Reactive.Bindings/Instance.uno

```diff
@@ -700,8 +731,9 @@ namespace Fuse.Reactive
 			//find last node prior to where we want to introduce
 			var lastNode = GetLastNodeFromIndex(windowIndex-1);
 
-			Parent.InsertNodes( Parent.Children.IndexOf(lastNode) + 1, newElements.GetEnumerator() );
+			//assign first for lookups during the rooting in Insert can find the data
 			wi.Nodes = newElements;
+			Parent.InsertNodes( Parent.Children.IndexOf(lastNode) + 1, newElements.GetEnumerator() );
```

Comment:
The ordering of the two statements is changed. It's the new `index` expression that requires this. Previously it didn't make a difference since nothing would callback into `Each` during the insertion.

It is a bugfix, but it only affects the new features.